ocaml/xapi-consts/api_errors.ml

```diff
@@ -100,6 +100,7 @@ let network_sriov_enable_failed = "NETWORK_SRIOV_ENABLE_FAILED"
 let network_sriov_disable_failed = "NETWORK_SRIOV_DISABLE_FAILED"
 let network_is_not_sriov_compatible = "NETWORK_IS_NOT_SRIOV_COMPATIBLE"
 let network_has_incompatible_sriov_pifs = "NETWORK_HAS_INCOMPATIBLE_SRIOV_PIFS"
+let vlan_network_incompatible = "VLAN_NETWORK_INCOMPATIBLE"
```

Comment:
Update this depending on how you update the error in `datamodel.ml`

Reply:
Done